drape_frontend/text_layout.cpp

```diff
@@ -124,15 +124,15 @@ class TextOutlinedGeometryGenerator
 
 void SplitText(strings::UniString & visText, buffer_vector<size_t, 2> & delimIndexes)
 {
-  char const * delims = " \n\t";
-  size_t count = visText.size();
+  char constexpr delims[] = " \n\t";
```

Comment:
nit: declare delims below, inside if near with delimsSize.
Otherwise there is a broken context wtf with -1. Next time somebody will refactor it with array of chars.